Products/Zuul/catalog/model_catalog.py

```diff
@@ -131,7 +131,7 @@ def model_index(self):
         return self._data_manager.model_index
 
     def _get_forbidden_classes(self):
-        return (Software, OperatingSystem)
+        return ()
```

Comment:
Software and Operating systems are MEProduct. If we don't index them we will not be able to get the instances for the related ProductClass